xclim/testing/tests/test_precip.py

```diff
@@ -92,6 +93,68 @@ def test_with_different_phases(self):
         np.testing.assert_array_almost_equal(out_liq + out_sol, out_tot, 4)
 
 
+class TestStandardizedPrecip:
+    nc_ds = os.path.join("sdba", "CanESM2_1950-2100.nc")
+
+    def test_3d_data_with_nans(self):
+        # test with data
+        ds = open_dataset(self.nc_ds).isel(location=1)
+        pr = ds.pr.sel(time=slice("2000"))  # kg m-2 s-1
+        prMM = pr
+        prMM *= 86400 * 1000
+        prMM.attrs["units"] = "mm/day"
```

Comment:
```suggestion
        prMM = convert_units_to(pr, "mm/day")
```
with `from xclim.core.units import convert_units_to` at the top of the file.